docs/release-notes/8.17.asciidoc

```diff
@@ -1,6 +1,26 @@
 [[release-notes-header-8.17.0]]
 == 8.17
 
+[discrete]
+[[release-notes-8.17.6]]
+=== 8.17.6
+
+[discrete]
+[[enhancements-8.17.6]]
+==== Enhancements
+* Allows {elastic-defend} users to opt out of event-driven Memory Protection scanning using the advanced policy ({kibana-pull}218354[#218354]).
+
+[discrete]
+[[bug-fixes-8.17.6]]
+==== Fixes
+* Fixes a bug that caused installed prebuilt detection rules to upgrade to their latest available versions when you installed a new {elastic-defend} integration or {agent} policy ({kibana-pull}217959[#217959]).
```

Comment:
```suggestion
* Fixes a bug that caused installed prebuilt detection rules to upgrade to their latest versions when you installed a new {elastic-defend} integration or {agent} policy ({kibana-pull}217959[#217959]).
```

Reply:
I do want to leave "available" here because it's an important qualifier in the sentence. My understanding is that there could be situations where the latest versions of a rule aren't available because users haven't updated their rules in a while.